spec/components/npq_separation/admin/statement_summary_component_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe NpqSeparation::Admin::StatementSummaryComponent, type: :component do
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::NumberHelper
+
+  subject(:rendered) { render_inline described_class.new(statement:) }
+
+  let(:statement) { create(:statement) }
+  let(:calculator) do
```

Comment:
Minor nit: Can you add a whitespace line before and after multiple blocks. You could then group the single line lets together